sysdump/sysdump.go

```diff
@@ -1448,6 +1452,53 @@ func (c *Collector) Run() error {
 		return fmt.Errorf("failed to close the worker pool: %w", err)
 	}
 
+	// Run each serial task in its own workerpool.
+	for i, t := range serialTasks {
```

Comment:
Should we execute the serialTasks before any other type?

Reply:
Might be a good idea in order to avoid triggering the Go runtime memory preallocation from completing the other tasks. I'll move them! :+1: